src/uwtools/strings.py

```diff
@@ -86,6 +86,7 @@ class STR:
     link: str = "link"
     mode: str = "mode"
     model: str = "model"
+    mpasinit: str = "mpas_init"
```

Comment:
This is weird. I have an mpas entry in my local copy, but not here. I'm not sure how to resolve that...digging in.

Reply:
Apparently, it was a brief fluke. I see it here now.